site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -299,6 +374,27 @@ paths:
                   application/json:
                      schema:
                         $ref: '#/components/schemas/Device'
+                     examples:
+                        fully-configured:
+                           summary: Fully configured device
+                           description: |
+                              The body contains the configuration properties of a device which
+                              has been configured extensively.
+                           value: |
+                              {
```

Comment:
The enabled property will not be included by our registry implementations if it hasn't been set explicitly during registration.
We could change this behavior, of course. See #2041

Reply:
But it can be specified when creating a device and we use the same examples AFAIK. I just thought that it'd be good that "fully configured" example shows it.

Reply:
Sure, I'll add it ...

Reply:
@dejanb I have added the *enabled* property and have also fixed the example response for credentials meta data only ...